#166265826 Set up and integrate Travis with README badge

encodedBicoding opened the pull request:
### What does this PR do?
Configures `.travis.yml` file to include scripts needed for travis to run. 
#### Description of Task to be completed?
1. Add git repo to travis using the GUI.
2. Create a `.travis.yml` file with configuration scripts in it.
3. Add travis badge to README.
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
This may not work properly now, as coverage script isn't available yet. 
Coverage script would be available
![Screenshot (52)](https://user-images.githubusercontent.com/41347407/58474871-4b253680-8144-11e9-8d8b-086451ec843c.png)
 when it has been set up by the assigned developer and merged.
#### What are the relevant pivotal tracker stories?
[#166265826](https://www.pivotaltracker.com/story/show/166265826)
#### Screenshots (if appropriate)
#### Questions:
N/A

chuxmykel committed:
chore(server setup): initialize express server with environment variables

- install dotenv and update server to run on port gotten from env variables
- add dev and start scripts for development and production environments
- create server folder and move all server related filed to the folder
- update README.md file with instructions on how to run the project

[Finishes #166265828]

chuxmykel committed:
chore(server setup): initialize express server with environment variables

-install dotenv and update server to run on port gotten from env variables
-add dev and start scripts for development and production environments
-create server folder and move all server related filed to the folder
-update README.md file with instructions on how to run the project

[Finishes #166265828]

cvjude committed:
chore(Tests): set up unit tests

Setup test enviroment using jest
Write test to ensure setup works

[Finishes #166265827]

cvjude committed:
Merge branch 'develop' into ch-setup-unit-testing-166265827

chuxmykel committed:
Merge branch 'develop' into ch/166265828/initialize-env-variables

encodedBicoding committed:
chore(travis):setup and integrate travis with README badge [(Delivers) #166265826]

encodedBicoding committed:
chore(travis):setup and integrate travis with README badge [(Delivers) #166265826]

chuxmykel commented on .travis.yml:
```diff
@@ -0,0 +1,11 @@
+node_js:
+  - "10.13.0"
+services:
+  - sequelize
```
Is this line really necessary?

encodedBicoding replied:
Which exactly?

encodedBicoding replied:
i thought i should add it there since we are going to be using the ORM. Correct me if i am wrong

chuxmykel replied:
[Check this out](https://docs.travis-ci.com/user/database-setup/#postgresql)

chuxmykel reviewed (dismissed):
I think this PR should wait for @cvjude's PR to be merged first so the tests can reflect

chuxmykel committed:
chore(server setup): initialize express server with environment variables
-install dotenv and update server to run on port gotten from env variables
-add dev and start scripts for development and production environments
-create server folder and move all server related filed to the folder
-update README.md file with instructions on how to run the project
[Finishes #166265828]

encodedBicoding committed:
Removes services keyfrom travis config

encodedBicoding reviewed (commented):
Done. Removed services key

emp-daisy committed:
Merge pull request #4 from emp-daisy/ch/166265828/initialize-env-variables

#166265828 Initialize express server

cvjude committed:
Merge branch 'develop' into ch-setup-unit-testing-166265827

emp-daisy committed:
Merge pull request #6 from emp-daisy/ch-setup-unit-testing-166265827

#166265827 set up unit tests

emp-daisy commented:
Rebase and fix conflicts

encodedBicoding committed:
removes services key from travis config

encodedBicoding committed:
Remove services key from travis config

encodedBicoding committed:
fix merge error in README.md

encodedBicoding committed:
Fix merge error in README.md

cvjude commented on package.json:
```diff
@@ -5,7 +5,12 @@
     "start": "nodemon --exec babel-node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --forceExit --verbose",
+    "coverage": "jest --forceExit --coverage && cat ./coverage/lcov.info | coveralls",
```
I think we have to change this script before we merge
It's running the tests twice
And its complaining that we don't have coveralls installed.

encodedBicoding replied:
I'm on it

encodedBicoding replied:
is that supposed to be my job? 
I didn't work on tests

encodedBicoding committed:
Remove coverage run script from travis.yml
